### DISCUSSION TITLE
Using DBeaver for Data Warehouseing

### DISCUSSION THREAD

**Discussion by CFM90** [dbeaver/dbeaver#10895] [Q&A]
Hello Devs and Community,

I recently discovered Dbeaver and I am impressed by its capabilities and "easy to get in" UX. I have a use case, but because of a lack of experience, I am not sure, if DBeaver fits it:

At my company we want to build a data warehouse and currently evaluate different commercial vendors like Oracle, Informatica or IBM (Gartner Leaders). I would prefer postgresql as a DB core engine and an open source/ cost efficient easy to use tool for ETL processes and automation (cron jobs). The amount of data is quite small -- every table is blow 10M rows and 25 col. and will not exceed 100M rows in the next 5 years. So I feel commercial vendors are an overkill and DBeaver could be an alternative (maybe Enterprise edition). I am not sure about sql background optimizations comm. vendors apply, but DBeaver does not. I am aware that vendors promote things besides ETL processing, mostly data catalogues etc. Furthermore we want to attach a reporting tool (Tableau, Power BI etc.), however this has nothing to do with DBeaver I guess. 

What is your opinion on using DBeaver for data warehouseing?

**Comment by p91paul** [comment]
Well the database tasks feature could be helpful achieving that. They can also be scheduled.
However I believe it means that you manually need to write all your queries (which is arguably faster than using those complicated gui commercial etl tools have, so not a big deal).
Actually, enterprise dbeaver also has a visual query builder AFAIK.
